packages/scss/src/mixins/_responsiveness.scss

```diff
@@ -0,0 +1,35 @@
+@mixin media_smaller_than($breakpoint) {
+	@if map-has-key(map-get($theme,"breakpoints"), $breakpoint) {
```

Comment:
Yes, but if the key doesn't exist, it throws a general error (stating that the key $breakpoint doesn't exist) without specifying the fact that it is a breakpoint that doesn't exist (error handled by these mixins)